.planning/codebase/TESTING.md

```
@@ -0,0 +1,298 @@
+      statements: 70,
+    },
+  },
+});
+```
+
+## Test Structure Pattern (If Implemented)
+
+Based on observed code organization:
+
+**File location:** Co-located next to implementation
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)